### DISCUSSION TITLE
List of Supported IMGMOUNT extensions?

### DISCUSSION THREAD

**Discussion by raekuul** [joncampbell123/dosbox-x#4574] [Q&A]
I recently updated to 2023.10.06 (I've skipped a few versions; the last version I was using was I think 2022.12.something); in prior versions IMGMOUNT would accept "games.hdd" as a fully valid filename and mount it without complaint (I do not know if it was mounting it with implied `-t hdd` but insofar as I could tell it was being done "correctly"), but with 2023.10.06 at least IMGMOUNT is now reporting "Unsupported extension .hdd: Mounted as raw IMG image."
Is there a list of the supported extensions for IMGMOUNT anywhere?

**Comment by rderooy** [comment]
https://dosbox-x.com/wiki/Guide%3AManaging-image-files-in-DOSBox%E2%80%90X#_harddisk_images

typically `.img` for plain mfm hdd images. But there is also support for vhd images.
